J/jlqml/build_tarballs.jl

```diff
@@ -2,16 +2,14 @@
 # `julia build_tarballs.jl --help` to see a usage message.
 using BinaryBuilder, Pkg
 
+julia_version = v"1.5.3"
+
 name = "jlqml"
-version = v"0.1.4"
+version = v"0.1.5"
```

Comment:
@barche Just to have it spelled out explicitly: you are OK with dropping support for Julia 1.3 and 1.4?

Reply:
Yes, this should be fine!